darshan-util/pydarshan/darshan/report.py

```diff
@@ -844,45 +791,21 @@ def mod_read_all_lustre_records(self, mod="LUSTRE", dtype=None, warnings=True):
 
 
         # fetch records
-        rec = backend.log_get_record(self.log, mod, dtype=dtype)
+        rec = backend.log_get_record(self.log, mod, dtype="numpy")
         while rec != None:
             self.records[mod].append(rec)
             self.data['modules'][mod]['num_records'] += 1
 
             # fetch next
-            rec = backend.log_get_record(self.log, mod, dtype=dtype)
+            rec = backend.log_get_record(self.log, mod, dtype="numpy")
 
 
         if self.lookup_name_records:
             self.update_name_records(mod=mod)
 
-        # process/combine records if the format dtype allows for this
-        if dtype == 'pandas':
-            combined_c = None
-
-            for rec in self.records[mod]:
-                obj = rec['counters']
-                #print(type(obj))
-                #display(obj)
-                
-                if combined_c is None:
-                    combined_c = rec['counters']
-                else:
-                    combined_c = pd.concat([combined_c, rec['counters']])
-                    
-
-            self.records[mod] = [{
-                'rank': -1,
-                'id': -1,
-                'counters': combined_c,
-                }]
-
-
-
 
 
-    def mod_records(self, mod, 
-                    dtype='numpy', warnings=True):
+    def mod_records(self, mod, dtype='numpy', warnings=True):
```

Comment:
Do we have a reason to keep the `dtype` parameter here? I can't find an instance where this method is actually used, but it could make sense to remove this one as well.